src/app/components/moj/moj-pagination/moj-pagination.component.html

```diff
@@ -0,0 +1,49 @@
+<nav class="moj-pagination" aria-label="Pagination navigation">
+  <app-moj-pagination-list>
+    <app-moj-pagination-item extraClasses="moj-pagination__item--prev">
+      @if (currentPage > 1) {
+        <app-moj-pagination-link
+          [changePage]="changePage"
+          [page]="currentPage - 1"
+          [currentPage]="currentPage"
+          [totalPages]="totalPages()"
+        >
+          Previous<span class="govuk-visually-hidden"> page </span></app-moj-pagination-link
+        >
+      }
+    </app-moj-pagination-item>
+    @for (page of pages(); track page) {
+      <app-moj-pagination-item [isActive]="page === currentPage">
+        <app-moj-pagination-link
+          [changePage]="changePage"
+          [page]="+page"
+          [currentPage]="currentPage"
+          [totalPages]="totalPages()"
+        >
+          @if (page !== '...') {
+            {{ page }}
+          }
+        </app-moj-pagination-link>
+        @if (page === '...') {
+          <span>…</span>
+        }
+      </app-moj-pagination-item>
+    }
+
+    <app-moj-pagination-item extraClasses="moj-pagination__item--next">
```

Comment:
Change this to this:
`<li app-moj-pagination-item class="moj-pagination__item--next">`